src/connectors/menu/connectMenu.ts

```diff
@@ -90,6 +106,14 @@ export type MenuRenderState = {
    * True if refinement can be applied.
    */
   canRefine: boolean;
+  /**
+   * Searches for values inside the list.
+   */
+  searchForItems(query: string): void;
+  /**
+   * `true` if the values are from an index search.
```

Comment:
We'll need to also update this in `connectRefinementList` (unrelated to this PR).

```suggestion
   * Whether the values are from an index search or not.
```